src/materials/nodes/NodeMaterial.js

```diff
@@ -362,11 +362,6 @@ class NodeMaterial extends Material {
 		 */
 		this.vertexNode = null;
 
-		/**
-		 * Uniforms used to pass data to the material.
-		 */
-		this.uniforms = {};
```

Comment:
`uniforms` should not be stored in `NodeMaterial`. This is a TODO in `PMREMGenerator` to remove the assignment.